src/Sdk/Cerbos/Sdk/Cloud/V1/Store/File.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021-2025 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+
+namespace Cerbos.Sdk.Cloud.V1.Store
+{
+    public sealed class File
+    {
+        private byte[] contents;
+        private string path;
+
+        private Api.Cloud.V1.Store.File R { get; set; }
+        public byte[] Contents {
+            get
+            {
+                return contents;
+            }
+        }
+        public string Path
+        {
+            get
+            {
+                return path;
+            }
+         }
+        public Api.Cloud.V1.Store.File Raw => R;
+
+        private File() {}
+
+        public File(Api.Cloud.V1.Store.File file)
+        {
+            R = file;
+            path = file.Path;
+            contents = file.Contents.ToByteArray();
+        }
+
+        public static File NewInstance()
```

Comment:
Given that both path and contents are mandatory, I'd just make `NewInstance` require both as parameters. The `With*` builder methods would just make it a lot more verbose than necessary, won't they?